ci/checks/style.sh

```diff
@@ -14,7 +14,7 @@ LANG=C.UTF-8
 source activate gdf
 
 # Run isort and get results/return code
-ISORT=`isort --recursive --check-only python`
+ISORT=`isort --check-only python/**/*.py`
```

Comment:
Any specific reason `isort --check-only python` is not used?

Reply:
> Any specific reason `isort --check-only python` is not used?

It didn't seem work locally when I did it.

Reply:
I think that might be environment-specific?

For me, it works. Plus with 5.0.0+ there is [cython support](https://timothycrosley.github.io/isort/docs/major_releases/introducing_isort_5/) for isort too, so we should be using `isort --check-only python` 


```bash
pgali@dt07:/nvme/0/pgali/cudf$ isort --check-only python/**/*.py
ERROR: /nvme/0/pgali/cudf/python/cudf/setup.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/dask_cudf/setup.py Imports are incorrectly sorted and/or formatted.

pgali@dt07:/nvme/0/pgali/cudf$ isort --check-only python
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/__init__.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/core/index.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/core/indexing.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/core/buffer.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/core/__init__.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/core/frame.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/core/dataframe.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/core/column_accessor.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/core/ops.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/core/multiindex.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/core/series.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/core/groupby/groupby.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/core/window/rolling.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/core/join/join.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/core/column/numerical.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/core/column/column.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/core/column/__init__.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/core/column/categorical.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/core/column/datetime.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/core/column/string.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_cuda/gpu.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/tests/test_dropna.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/tests/utils.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/tests/test_dataframe_copy.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/tests/test_reshape.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/tests/test_repr.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/tests/test_duplicates.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/tests/test_joining.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/tests/test_udf_binops.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/tests/test_index.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/tests/test_datetime.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/tests/test_monotonic.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/tests/test_string.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/tests/test_hdfs.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/tests/test_dataframe.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/tests/conftest.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/tests/test_column.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/comm/gpuarrow.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/datetime.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/stream_compaction.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/copying.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/join.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/concat.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/merge.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/utils.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/orc.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/avro.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/rolling.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/reshape.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/__init__.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/dlpack.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/groupby.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/json.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/csv.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/hash.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/null_mask.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/column.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/parquet.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/scalar.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/aggregation.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/transform.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/reduce.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/filling.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/table.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/types.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/sort.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/transpose.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/partitioning.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/unary.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/gpuarrow.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/search.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/string_casting.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/quantiles.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/replace.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/binaryop.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/nvtx/nvtx.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/nvtx/_lib/__init__.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/strings/strip.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/strings/substring.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/strings/combine.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/strings/attributes.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/strings/findall.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/strings/contains.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/strings/find.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/strings/case.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/strings/find_multiple.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/strings/padding.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/strings/capitalize.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/strings/char_types.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/strings/translate.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/strings/wrap.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/strings/extract.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/strings/replace.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/strings/replace_re.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/strings/convert/convert_urls.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/strings/split/split.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/strings/split/partition.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/nvtext/ngrams_tokenize.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/nvtext/tokenize.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/nvtext/generate_ngrams.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/nvtext/replace.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/nvtext/normalize.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/_lib/io/utils.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/io/__init__.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/io/json.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/io/csv.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/io/orc.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/io/parquet.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/io/avro.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/utils/utils.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/utils/cudautils.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/utils/applyutils.py Imports are incorrectly sorted and/or formatted.
ERROR: /nvme/0/pgali/cudf/python/cudf/cudf/utils/gpu_utils.py Imports are incorrectly sorted and/or formatted.
```

Reply:
Hmmm, I experimented a bit and it seems like if I add `--atomic` to the CI script it will work.

Reply:
Hmm -- what do you see when running `isort --check-only python`? It seems to work for me:

```bash

ashwint@superunknown ~/w/g/r/cudf (list-dtype)> isort -V                                                                                                                                             (cudf_dev)

                 _                 _
                (_) ___  ___  _ __| |_
                | |/ _/ / _ \/ '__  _/
                | |\__ \/\_\/| |  | |_
                |_|\___/\___/\_/   \_/

      isort your imports, so you don't have to.

                    VERSION 5.0.4

ashwint@superunknown ~/w/g/r/cudf (list-dtype)> isort --check-only python &| head                                                                                                                    (cudf_dev)
ERROR: /home/ashwint/workspace/github/rapidsai/cudf/python/cudf/cudf/__init__.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/ashwint/workspace/github/rapidsai/cudf/python/cudf/cudf/comm/gpuarrow.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/ashwint/workspace/github/rapidsai/cudf/python/cudf/cudf/core/__init__.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/ashwint/workspace/github/rapidsai/cudf/python/cudf/cudf/core/index.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/ashwint/workspace/github/rapidsai/cudf/python/cudf/cudf/core/series.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/ashwint/workspace/github/rapidsai/cudf/python/cudf/cudf/core/dataframe.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/ashwint/workspace/github/rapidsai/cudf/python/cudf/cudf/core/multiindex.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/ashwint/workspace/github/rapidsai/cudf/python/cudf/cudf/core/column_accessor.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/ashwint/workspace/github/rapidsai/cudf/python/cudf/cudf/core/indexing.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/ashwint/workspace/github/rapidsai/cudf/python/cudf/cudf/core/buffer.py Imports are incorrectly sorted and/or formatted.
ashwint@superunknown ~/w/g/r/cudf (list-dtype)>
```

Reply:
So to be clear, we need all of the below to be consistent:
* CI script, which is currently: `isort --check-only python/**/*.py`
* Git hook, which is currently: `isort`
* Documentation for local, which is curently: `isort --atomic python/**/*py`

I am investigating why the 3 above seem to be in sync. If I replace the CI script and local `isort` with Python, things start being inconsistent (meaning I expect once I locally format with `isort`, both the git commit hook and the CI script should pass).

Reply:
@shwina I get something similar:
```
(cudf_dev)  ✘ cph@z8-g4  ~/cudf   update-isort  isort -V 

                 _                 _
                (_) ___  ___  _ __| |_
                | |/ _/ / _ \/ '__  _/
                | |\__ \/\_\/| |  | |_
                |_|\___/\___/\_/   \_/

      isort your imports, so you don't have to.

                    VERSION 5.0.4

(cudf_dev)  cph@z8-g4  ~/cudf   update-isort  isort --check-only python &| head
ERROR: /home/cph/cudf/python/dask_cudf/setup.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/cph/cudf/python/dask_cudf/dask_cudf/backends.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/cph/cudf/python/dask_cudf/dask_cudf/core.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/cph/cudf/python/dask_cudf/dask_cudf/sorting.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/cph/cudf/python/dask_cudf/dask_cudf/io/opt_parquet.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/cph/cudf/python/dask_cudf/dask_cudf/io/orc.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/cph/cudf/python/dask_cudf/dask_cudf/io/tests/test_parquet.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/cph/cudf/python/dask_cudf/dask_cudf/io/tests/test_json.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/cph/cudf/python/dask_cudf/dask_cudf/io/tests/test_csv.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/cph/cudf/python/dask_cudf/dask_cudf/io/tests/test_orc.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/cph/cudf/python/dask_cudf/dask_cudf/io/tests/test_s3.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/cph/cudf/python/dask_cudf/dask_cudf/tests/test_accessor.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/cph/cudf/python/dask_cudf/dask_cudf/tests/test_groupby.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/cph/cudf/python/dask_cudf/dask_cudf/tests/test_binops.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/cph/cudf/python/dask_cudf/dask_cudf/tests/test_sort.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/cph/cudf/python/dask_cudf/dask_cudf/tests/test_distributed.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/cph/cudf/python/dask_cudf/dask_cudf/tests/test_core.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/cph/cudf/python/dask_cudf/dask_cudf/tests/test_onehot.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/cph/cudf/python/dask_cudf/dask_cudf/tests/test_join.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/cph/cudf/python/dask_cudf/dask_cudf/tests/test_reductions.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/cph/cudf/python/dask_cudf/dask_cudf/tests/test_delayed_io.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/cph/cudf/python/custreamz/custreamz/tests/test_dataframes.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/cph/cudf/python/cudf/setup.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/cph/cudf/python/cudf/cudf/__init__.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/cph/cudf/python/cudf/cudf/_lib/datetime.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /home/cph/cudf/python/cudf/cudf/_lib/groupby.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /home/cph/cudf/python/cudf/cudf/_lib/utils.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /home/cph/cudf/python/cudf/cudf/_lib/parquet.pyx Imports are incorrectly sorted and/or formatted.
ERROR: /home/cph/cudf/python/cudf/cudf/_lib/avro.pyx Imports are incorrectly sorted and/or formatted.
```
However, if I do:
`(cudf_dev)  ✘ cph@z8-g4  ~/cudf   update-isort  isort --check-only python/**/*.py `
I don't get any changes. And this is the only setting I have found to be consistent with the CI script (see msg above)

Reply:
> However, if I do:
> `(cudf_dev) ✘ cph@z8-g4  ~/cudf   update-isort  isort --check-only python/**/*.py `
> I don't get any changes. And this is the only setting I have found to be consistent with the CI script (see msg above)

This is because you are also restricting check to only `.py` files. What is the output for CI ? I see that CI just runs the command we give in `ci/checks/stype.sh`, so is giving `isort --check-only python` in that file not working ?

Reply:
That's weird -- the two give me identical results 😕 

```
ashwint@superunknown ~/w/g/r/cudf (list-dtype) [1]> isort --check-only python/**/*.py &| head                                                                                                        (cudf_dev)
ERROR: /home/ashwint/workspace/github/rapidsai/cudf/python/cudf/cudf/core/column/categorical.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/ashwint/workspace/github/rapidsai/cudf/python/cudf/cudf/core/column/column.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/ashwint/workspace/github/rapidsai/cudf/python/cudf/cudf/core/column/datetime.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/ashwint/workspace/github/rapidsai/cudf/python/cudf/cudf/core/column/numerical.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/ashwint/workspace/github/rapidsai/cudf/python/cudf/cudf/core/column/string.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/ashwint/workspace/github/rapidsai/cudf/python/cudf/cudf/core/dataframe.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/ashwint/workspace/github/rapidsai/cudf/python/cudf/cudf/core/frame.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/ashwint/workspace/github/rapidsai/cudf/python/cudf/cudf/core/groupby/groupby.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/ashwint/workspace/github/rapidsai/cudf/python/cudf/cudf/core/index.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/ashwint/workspace/github/rapidsai/cudf/python/cudf/cudf/core/indexing.py Imports are incorrectly sorted and/or formatted.

ashwint@superunknown ~/w/g/r/cudf (list-dtype)> isort --check-only python &| head                                                                                                                    (cudf_dev)
ERROR: /home/ashwint/workspace/github/rapidsai/cudf/python/cudf/cudf/__init__.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/ashwint/workspace/github/rapidsai/cudf/python/cudf/cudf/comm/gpuarrow.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/ashwint/workspace/github/rapidsai/cudf/python/cudf/cudf/core/__init__.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/ashwint/workspace/github/rapidsai/cudf/python/cudf/cudf/core/index.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/ashwint/workspace/github/rapidsai/cudf/python/cudf/cudf/core/series.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/ashwint/workspace/github/rapidsai/cudf/python/cudf/cudf/core/dataframe.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/ashwint/workspace/github/rapidsai/cudf/python/cudf/cudf/core/multiindex.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/ashwint/workspace/github/rapidsai/cudf/python/cudf/cudf/core/column_accessor.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/ashwint/workspace/github/rapidsai/cudf/python/cudf/cudf/core/indexing.py Imports are incorrectly sorted and/or formatted.
ERROR: /home/ashwint/workspace/github/rapidsai/cudf/python/cudf/cudf/core/buffer.py Imports are incorrectly sorted and/or formatted.
```

Reply:
> That's weird -- the two give me identical results 

What is in your `style.sh`?

Reply:
@shwina @galipremsagar and I had a call. It looks like the issue is without the `python/**/*.py` isort fails to pick up the setup.cfg file. Looking into it now. This PR can be a temporary solution worst case.